scripts/release-binary.sh

```diff
@@ -82,7 +87,7 @@ if [ "${CHECK}" -eq 1 ]; then
     [[ "${UBUNTU_RELEASE}" == 'xenial' ]] || { echo 'Must run on Ubuntu 16.04 (Xenial).'; exit 1; }
   fi
   [[ "$(uname -m)" == 'x86_64' ]] || { echo 'Must run on x86_64.'; exit 1; }
-elif [ -n "${DST}" ]; then
+elif [ -n "${DST}" ] && [ "${BUILD_FOR_CENTOS}" -ne 0 ]; then
```

Comment:
Since you're explicitly marking builds for CentOS, you should perform the check to make sure it's running on CentOS (i.e. provide CentOS equivalent to the regular check), and then you can remove `-i` from the `Makefile` targets.